### PR TITLE
Links: use https for some urls

### PR DESCRIPTION
**Pull request by imba-tjd**
These links use `http` in `Readme`, I `find and replace` them. If it were not appropriate, I would restrict this in `.md`.

```
http://stackoverflow.com
http://www.typescriptlang.org
http://nuget.org
http://nodejs.org
```

### CODE REVIEW

**Comment by imba-tjd** [comment]
```
Error in needle/v0
Error in needle
```

**Comment by peterblazejewicz** [comment]
There could be a package, which was not updated recently, so now when run with CI tests, one can get unexpected errors. First to be sure it's not yours, rebase your pull requests against most recent master.
I'd advise, if problem persist, remove `types/node` package changes and revisit later

**Comment by imba-tjd** [comment]
This is based on latest master.

**Comment by imba-tjd** [comment]
I don't know what to do.
Shall I close this and open another PR only touches md?

File: types/angular/index.d.ts [hunk 1]
```diff
@@ -316,7 +316,7 @@ declare namespace angular {
         /**
          * this is necessary to be able to access the scoped attributes. it's not very elegant
          * because you have to use attrs['foo'] instead of attrs.foo but I don't know of a better way
-         * this should really be limited to return string but it creates this problem: http://stackoverflow.com/q/17201854/165656
+         * this should really be limited to return string but it creates this problem: https://stackoverflow.com/q/17201854/165656
```
Comment (peterblazejewicz) [inline types/angular/index.d.ts:319]:
```suggestion
         * this should really be limited to return string but it creates this problem: http://stackoverflow.com/q/17201854/165656
```
revert this change for now, it runs tests for related packages for `angular`, and those are quite outdated. Types augmentation in 3rd party types fails under TS 4.5

**Review by peterblazejewicz — APPROVED** [review]
LGTM! (this was reviewed locally using char diff)
@imba-tjd thx!
you can revisit `angular` types change in seprate PR, so it could be easier to exempt from CI checks

**Comment by imba-tjd** [comment]
Ready to merge

**Pull request merged**